.openshift-ci/debug-tips.md

```diff
@@ -0,0 +1,21 @@
+# Debug OpenShift CI jobs
+
+## Download all artifacts
+
+1. Visit the jobs page
+2. Click on the `Aritfacts` link
+3. Append `artifacts/merge-gke-nongroovy-e2e-tests/stackrox-stackrox-e2e-test/artifacts/howto-locate-other-artifacts-summary.html` to the URL openend in your browser.
+4. Copy the `gsutil` command and run it locally to copy the job artifacts. Alternatively you can browse them in GCP.
+
+### Working with the artifacts
```

Comment:
The commands look alright, but I don't have experience with the StackRox specifics in DB restore, maybe one of you @stackrox/infra ?